pkg/controller/kfdef/kfdef_controller.go

```diff
@@ -89,14 +85,14 @@ func watchKubeflowResources(c controller.Controller) error {
 		})
 		err := c.Watch(&source.Kind{Type: u}, &handler.EnqueueRequestsFromMapFunc{
 			ToRequests: handler.ToRequestsFunc(func(a handler.MapObject) []reconcile.Request {
-				log.Info("watch a change for kfdef resource: %s.%s", a.Meta.GetName(), a.Meta.GetNamespace())
+				log.Infof("watch a change for kfdef resource: %v.%v.", a.Meta.GetName(), a.Meta.GetNamespace())
 				return []reconcile.Request{
 					{NamespacedName: kfdefSingletonNN},
 				}
 			}),
 		}, ownedResourcePredicates)
 		if err != nil {
-			log.Info("cannot create watch for resources ", t.Kind, t.Group, t.Version, ". Error: ", err)
+			log.Infof("cannot create watch for resources %v %v/%v. Error: %v.", t.Kind, t.Group, t.Version, err)
```

Comment:
Sorry, previously the other logger library wasn't letting me to pass certain error message so that made me used info. Can you change the log level here to Error? thanks.